sr

```diff
@@ -31,11 +31,11 @@ case "$COMMAND" in
             -sdk "${SDK}" \
             -tools-directory "${SDK}/usr/bin" \
             -Xcc --sysroot="${NDK}/sysroot" \
+            -Xlinker -lz \
```

Comment:
i just saw this commit in the history and "terrible hack" got my attention :smile: 
https://github.com/flowkey/swift-android-toolchain/pull/6/commits/6d6cfbb4837ab803c1efa63aaa12dfe3710d8639

do you think we should clean this up before merge @ephemer ?